tests/ewasm_tests/yul.rs

```diff
@@ -1,117 +0,0 @@
-// SPDX-License-Identifier: Apache-2.0
-
-use crate::build_solidity;
-use byte_slice_cast::AsByteSlice;
-use ethabi::{Token, Uint};
-
-#[test]
-fn eth_builtins() {
-    let mut runtime = build_solidity(
```

Comment:
The builtins tested here are not tested elsewhere. There need to be tests for them on another target, depending on their compatibility.

Reply:
We can't do runtime tests for the builtins anymore for ewasm/evm because we don't have codegen. So, I've added the tests to `tests/evm.rs`

Reply:
This is the only place in Solang where I test `address`, `selfbalance`, `caller`, `callvalue`, and `extcodesize`. By removing these runtime tests, there will be no test for them at all.

`address`, `caller`, `callvalue` and `selfbalance` are available for both Solana and Substrate. Instead of deleting the test, please, move it to a Solana or a Substrate runtime test. Only `extcodesize` is not available for those targets.

Reply:
I had missed that, thanks for pointing that out.